contributors/devel/sig-scalability/hollow-node-sts.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hollow-node
+  namespace: kubemark
+spec:
+  clusterIP: None
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    name: hollow-node
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Why stateful set?

Reply:
In our production, network resource and  podName are tied together.
Avoiding to allocate and recycle ips frequently, we like to use statefulset to deploy pods which can have constant names and constant ips.

In fact, we can use  sts, deploy, rc to deploy these hollow-nodes here.

Reply:
Aah, that's reasonable then. I was asking because we already have an RC-based kubemark template here - https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/resources/hollow-node_template.yaml. So maybe we could've just reused that.

Reply:
Done, but also keep the template as a simplified version.